localstack/services/awslambda/lambda_starter.py

```diff
@@ -34,6 +36,11 @@ def start_lambda(port=None, asynchronous=False):
         endpoint=handle_lambda_url_invocation,
     )
 
+    if not config.LAMBDA_EXECUTOR:
+        default_mode = get_default_executor_mode()
+        LOG.debug("No executor set. Lambda falling back to default executor mode: %s", default_mode)
+        log.event("lambda:config", version="v1", default_executor_mode=default_mode)
```

Comment:
should we maybe log this in any case? i know we're already tracking the config.LAMBDA_EXECUTOR variable through the config variables. but this seems more consistent to me.

```suggestion
 
  log.event("lambda:config", version="v1", executor_mode=config.LAMBDA_EXECUTOR, default_executor_mode=get_default_executor_mode())
```

Reply:
@ackdav  do you think either version would make this easier/harder for us to process later in our analytics?

Reply:
Great question!
I would prefer @thrau 's version for now. While this creates a tiny bit of duplicate value (executor_mode is logged in both startup as well as here), the benefit is that we have it here in the same payload. This makes analyzing specifically executor_mode a bit easier post hoc.
Btw - great point regarding versioning it!